README.md

```diff
@@ -41,22 +43,51 @@ nut_influx_connector \
     [OPTIONS ...]
 ```
 
+### MQTT Output Example
+
+```text
+nut_influx_connector \
+    -mqtt-server tcp://192.168.1.1:1883 \
+    -mqtt-topic "home/UPS" \
+    -mqtt-username "mqtt_user" \
+    -mqtt-password "mqtt_pass" \
+    -ups deskups \
+    -ups-nametag "work_desk" \
+    [OPTIONS ...]
+```
+
+This will publish measurements to topics like `home/UPS/battery_charge_percent`, `home/UPS/power`, etc.
+
 ## Options
 
-* `-heartbeat-url string`: URL to GET every 60s, URL to GET every 60s, if and only if the program has successfully sent NUT statistics to Influx in the past 120s.
-- `-influx-bucket string`: InfluxDB bucket. Supply a string in the form `database/retention-policy`. For the default retention policy, pass just a database name (without the slash character). Required.
-- `-influx-password string`: InfluxDB password.
-- `-influx-server string`: InfluxDB server, including protocol and port, e.g. `http://192.168.1.4:8086`. Required.
-- `-influx-timeout int`: Timeout for writing to InfluxDB, in seconds. (default `3`)
-- `-influx-username string`: InfluxDB username.
-- `-measurement-name string`: InfluxDB measurement name. (default `ups_stats`)
-- `-poll-interval int`: Polling interval, in seconds. (default `30`)
-- `-print-usage`: Log energy usage (in watts) to standard error.
+### General Options
+
 - `-ups string`: UPS to read status from, format `upsname[@hostname[:port]]`. Required.
 - `-ups-nametag string`: Value for the `ups_name` tag in InfluxDB. Required.
+- `-poll-interval int`: Polling interval, in seconds. (default `30`)
+- `-print-usage`: Log energy usage (in watts) to standard error.
+- `-heartbeat-url string`: URL to GET every 60s, if and only if the program has successfully sent NUT statistics to output in the past 120s.
 - `-help`: Print help and exit.
 - `-version`: Print version and exit.
 
+### InfluxDB Options
+
+- `-influx-server string`: InfluxDB server, including protocol and port, e.g. `http://192.168.1.4:8086`.
+- `-influx-bucket string`: InfluxDB bucket. Supply a string in the form `database/retention-policy`. For the default retention policy, pass just a database name (without the slash character).
+- `-influx-username string`: InfluxDB username.
+- `-influx-password string`: InfluxDB password.
+- `-influx-timeout int`: Timeout for writing to InfluxDB, in seconds. (default `3`)
+- `-measurement-name string`: InfluxDB measurement name. (default `ups_stats`)
+
+### MQTT Options
+
+- `-mqtt-server string`: MQTT broker server, including protocol and port, e.g. `tcp://192.168.1.1:1883`.
```

Comment:
Please verify whether the MQTT library actually requires the protocol `tcp://` to be specified. I suspect it does not. If not, update all documentation.